### PR TITLE
Add OpenSans and Flaticon fonts

### PR DESCRIPTION
**Pull request by zrebcu411**

### CODE REVIEW

File: static/fonts/flaticon/flaticon.html [hunk 1]
```diff
@@ -0,0 +1,575 @@
+<!DOCTYPE html>
```
Comment (FieryCod) [inline static/fonts/flaticon/flaticon.html]: Do we need this file?

File: static/fonts/flaticon/flaticon.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
+	/*
```
Comment (FieryCod) [inline static/fonts/flaticon/flaticon.css]: Do we need this file? No we don't

File: static/fonts/flaticon/_flaticon.scss [hunk 3]
```diff
@@ -0,0 +1,87 @@
+    /*
```
Comment (FieryCod) [inline static/fonts/flaticon/_flaticon.scss]: Please fix indent and move variables to another file

File: nuxt.config.js [hunk 4]
```diff
@@ -11,7 +11,9 @@ module.exports = {
       { hid: 'description', name: 'description', content: 'Nuxt.js project' }
     ],
     link: [
-      { rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' }
+      { rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' },
+      { rel: 'stylesheet', href: 'https://fonts.googleapis.com/css?family=Open+Sans:300,400&amp;subset=latin-ext' },
+      { rel: 'stylesheet', href: '/fonts/flaticon/flaticon.css' }
```
Comment (FieryCod) [inline nuxt.config.js]: We got a section with css assets, please move there import of flaticon

File: static/fonts/flaticon/flaticon.html (hunk 1 above)
Comment (zrebcu411) [reply static/fonts/flaticon/flaticon.html]: Yes, we need this file for development purposes. This is a demo file. How will you recognize which class use for a specific icon?

Comment (FieryCod) [reply static/fonts/flaticon/flaticon.html]: Ok, thanks for clarification :)

File: static/fonts/flaticon/_flaticon.scss (hunk 3 above)
Comment (zrebcu411) [reply static/fonts/flaticon/_flaticon.scss]: No, an icon pack will be replaced a lot of times. There is no sense to moving variables to another file on every new icon added (adding new icon generates totally new files)

Comment (FieryCod) [reply static/fonts/flaticon/_flaticon.scss]: Ok I see so we can merge your changes :)

File: nuxt.config.js (hunk 4 above)
Comment (zrebcu411) [reply nuxt.config.js]: Good point, this will be a better place for flaticon import :)

**Pull request merged**